index.html

```diff
@@ -24,10 +24,10 @@
         <section class="main-section">
             <h2>
                 Forrest Adventure
-                <img src="./assets/chipmunk.png">
-                <img src="./assets/skunk.png">
-                <img src="./assets/racoon.png">
-                <img src="./assets/squirrel.png">
+                <img src="../assets/chipmunk.png">
```

Comment:
for these try: '/assets/chipmunk.png' or './assets/chipmunk.png' for all of the images

Reply:
I realize you already had this, what I did was fork your repo, and tried to change it to get it redeployed.  So this is what is working.  Also, at the end of your index.html for your 'home page' try making the path for the 'app.js' to './app.js'